config/routes.rb

```diff
@@ -1,11 +1,6 @@
 Rails.application.routes.draw do
   root to: 'dashboards#index', via: :get
   resource :dashboard, only: [:show]
-
-  get '/login', to: 'sessions#new'
-  post '/login', to: 'sessions#create'
-  get '/logout', to: 'sessions#destroy'
-
   resources :snippets, only: [:new, :create]
-  resources :users, only: [:new, :create]
+  resources :users, controller: :users, only: [:create]
```

Comment:
Nope!

In this case the `UsersController` inherits from the controller inside Clearance, and we want to override that inheritance for just the `create` action, hence the explicit controller there. 

However, I removed it and it still works so 🙌